<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneralPage</class>
 <widget class="QWidget" name="GeneralPage">
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <spacer name="verticalSpacer_1">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QFrame" name="verticalFrame">
     <layout class="QVBoxLayout" name="verticalLayout_General">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <underline>false</underline>
         </font>
        </property>
        <property name="text">
         <string>General</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_1">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_2">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::FieldsStayAtSizeHint</enum>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>UI font</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QFontComboBox" name="uiFontFamily"/>
          </item>
          <item>
           <widget class="QComboBox" name="uiFontPointSize">
            <property name="editable">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="closeToTray">
        <property name="toolTip">
         <string>Make close button [X] minimize to tray instead of closing main window.</string>
        </property>
        <property name="text">
         <string>Close to tray</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="verticalFrame_2">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_Timeline">
      <item alignment="Qt::AlignLeft">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <underline>false</underline>
         </font>
        </property>
        <property name="text">
         <string>Timeline</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::FieldsStayAtSizeHint</enum>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Layout</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="timeline_layout"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Font</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QFontComboBox" name="timelineFontFamily"/>
          </item>
          <item>
           <widget class="QComboBox" name="timelineFontPointSize">
            <property name="editable">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="useShuttleScrollbar">
        <property name="toolTip">
         <string>Control scroll velocity instead of position with the timeline scrollbar. Requires restart.</string>
        </property>
        <property name="text">
         <string>Use shuttle scrollbar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="loadFullSizeImages">
        <property name="toolTip">
         <string>Automatically download a full-size image instead of a thumbnail.</string>
        </property>
        <property name="text">
         <string>Load full-size images</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QFrame" name="verticalFrame_3">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Notifications</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="fullNotif">
          <property name="toolTip">
           <string>Show notifications and activate the window.</string>
          </property>
          <property name="text">
           <string>Full</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="gentleNotif">
          <property name="toolTip">
           <string>Show notifications but do not activate the window.</string>
          </property>
          <property name="text">
           <string>Non-intrusive</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="noNotif">
          <property name="toolTip">
           <string>Notifications are entirely suppressed.</string>
          </property>
          <property name="text">
           <string>Highlight only</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_exp">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
